### PR TITLE
Fix space position in line break

### PR DESCRIPTION
**Pull request by gorjan-h4**
Import original linebreak pairs. Fix the spacePos parameter by resetting it when a space->non-space transition is detected.

- Added the original linebreaker tests.
- Also added property tests to validate that space designations contain characters.

### CODE REVIEW

File: tests/unicode.util.ts [hunk 1]
```diff
@@ -0,0 +1,159 @@
+//these tests are weird, possibly incorrect or just tailored differently. we skip them.
+
+export let skipTests = [
```
Comment (spion) [inline tests/unicode.util.ts:3]: not sure how they are weird, could you clarify by example?

File: tests/string-gen.util.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    aLength != null
+      ? bLength != null
+        ? fc.array(charArb, aLength, bLength)
+        : fc.array(charArb, aLength)
+      : fc.array(charArb);
+  return arrayArb.map(tab => tab.join(''));
+}
+
+function spacedArbitrary(charArb: fc.Arbitrary<string>, minLength?: number, maxLength?: number) {
+  const spaceChars = fc.frequency({ arbitrary: charArb, weight: 100 }, { arbitrary: fc.constant(' '), weight: 1 });
```
Comment (spion) [inline tests/string-gen.util.ts]: Hmm I would expect more of a 15:1 ratio rather than 100:1?

Comment (gorjan-h4) [reply tests/string-gen.util.ts]: Fixed

File: tests/unicode.util.ts (hunk 1 above)
Comment (gorjan-h4) [reply tests/unicode.util.ts:3]: From original line breaker, not sure how they're weird either...

**Pull request merged**